shiny/observatory/shinytester.R

```diff
@@ -4,28 +4,19 @@ shinyTester <- tabPanel(
  sidebarLayout(
```

Comment:
Rename and retitle this file even if it is just "Home"

Reply:
ok